Doc/library/sqlite3.rst

```diff
@@ -2440,11 +2440,15 @@ Here's an example of a factory that return a :class:`~collections.namedtuple`.
    >>> row.b   # Attribute access.
    2
 
+As an optimisation, decorate :func:`!_make_cls` with
+:func:`!functools.lru_cache` to avoid creating multiple named tuples
+with identical column spec.
+See :func:`functools.lru_cache` for more details.
```

Comment:
Here is the link to `lru_cache`!  Since you have it here, then it's ok to use the `!` above, but also consider removing this sentence and just have the link above.

Reply:
```suggestion
```

I'd already added it above, and I also agree it is redundant to mention it again here, so this implements your recommendation of just eliding this line.

Reply:
Yes, I deliberately added the link because of the "See ... for more details" sentence[^1]. However, I'll reword this anyways.

[^1]: that sentence would have been very weird without a link...